src/ui/page/internalError/internalError.scss

```diff
@@ -0,0 +1,131 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins.scss';
+
+.okp4-dataverse-portal-internal-error-page-main {
+  @include with-theme() {
+    height: 100%;
+
+    &.dark {
+      background: center / cover url('@/ui/asset/images/internal-error-dark.webp');
+
+      @include use-breakpoints(('mobile')) {
+        background: unset;
+      }
+
+      @include use-breakpoints(('tablet')) {
+        background-size: contain;
+        background-repeat: no-repeat;
+      }
+
+      .okp4-dataverse-portal-internal-error-page-text-wrapper {
+        @include use-breakpoints(('mobile')) {
+          background: center url('@/ui/asset/images/internal-error-dark-mobile.webp') no-repeat;
+        }
+      }
+    }
+
+    &.light {
```

Comment:
I think you can avoid code repetition in `dark` and `light` classes

Reply:
Done!